code/modules/industrial_lift/elevator/elevator_panel.dm

```diff
@@ -93,13 +80,21 @@
 
 	// This is exclusively for linking in mapload, just to ensure all elevator parts are created,
 	// and also so we can throw mapping errors to let people know if they messed up setup.
+	link_with_lift(log_error = TRUE)
+
+/// Link with associated lift objects, only log failure to find a lift in LateInit because those are mapped in
+/obj/machinery/elevator_control_panel/proc/link_with_lift(log_error)
```

Comment:
```suggestion
/obj/machinery/elevator_control_panel/proc/link_with_lift(log_error = FALSE)
```

so it remains "typed" as a boolean rather than just be a null thing that we set TRUE sometimes